Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -268,124 +272,78 @@ - (void)videoPlaybackStarted:(NSDictionary *)properties withOptions:(NSDictionar
 
     NSDictionary *map = @{
         @"ns_st_mp" : properties[@"video_player"] ?: @"*null",
+        @"ns_st_ci" : properties[@"content_asset_id"] ?: @"0"
     };
 
-    [self.streamAnalytics createPlaybackSessionWithLabels:map];
-
-    // The label ns_st_ci must be set through a setAsset call
```

Comment:
Looks like previously we has this set through a `setAsset` call. I can't find the documentation about this property in the docs. Can you direct me to the documentation of id we can remove this from this call?

Reply:
It's on[ page 28 ](https://app.box.com/s/csd2t9sb9z0m6gdadg852ysl4724jac8/file/545661486639).